### DISCUSSION TITLE
Gestione rete e domotica

### DISCUSSION THREAD

**Discussion by stefano196710** [indomus/forum#793] [HUB: Home Assistant]
Ciao a tutti.
Seguendo le guide su Internet e di questo sito, da quasi 1 anno, ho la domotica in funzione su Raspberry Pi 3 B+ con Home Assistant OS.
Sono riuscito ad integrare Sonoff con Tasmota e MQTT, quindi funziona tutto come previsto.

Nel tempo ho notato 2 problemi:
Il Raspberry sempre "caldo" con la CPU da 60°C, fino al massimo di 70°C
Da quando ho installato la domotica, capita 5/6 volte al giorno, che il router perde la connessione ad Internet e con i dispositivi per 30 secondi, può essere causato dalla continua comunicazione della domotica con Raspberry?

Il mio obiettivo sarebbe quello di aggiornare i dati solo quando qualcuno accede ad HA e di ottimizzare tutto il sistema, per renderlo meno "pesante" per il router

Grazie

**Comment by snakuzzo** [comment]
Sulla temperatura...credo sia normale. Prova a dissipare/raffreddare.
Quanto alla domanda sulle cause della perdita di connessione ad internet, non c'entrano nulla i device che hai in locale.
HA (così come altri hub personali) va bene così com'è e non è necessario inventarsi nulla di quello che dici...in primis perché non c'entra niente coi tuoi problemi di rete e poi perché non avrebbe alcun senso (tutte le cose automatiche lato server/device finirebbero nel secchio)

**Comment by stefano196710** [comment]
Ciao, grazie per la risposta.
Per la temperatura vedrò d'acquistare un dissipatore passivo migliore.

Avevo pensato ad un problema di HA, visto che le disconnessioni avvengono da quando ho la domotica in funzione.
Vedrò di aggiornare il router con un modello più recente, per verificare se è quello il problema.

Mentre sul lato ottimizzazione domotica, avevo letto diversi consigli.
Per esempio, un'installazione a parte è più efficiente, cioè HA sopra un sistema operativo (tipo Ubuntu) o varie configurazioni degli addons ed ecc, ma non ho capito se ci sono benefici tangibili o meno.
Per esempio il mio HA OS è a 32bit, anche se consigliano la 64, non capisco le differenze

**Comment by dompa72** [comment]
per il dissipatore ho utilizzato questo https://www.amazon.it/gp/product/B081R54N7Z/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

per sicurezza hai configurato l'accesso ad HA con connessioni protette (https)?
Se no, un passaggio su questa guida lo farei 
https://indomus.it/guide/come-installare-e-configurare-caddy-con-docker-su-raspbian-di-raspberry-pi/

attenzione alla SD, di norma dura 2 anni.... consiglio di fare dei backup

32 o 64 bit è una differenza di architettura della CPU.
Installazione su UBUNTU dipende sempre da cosa concedi ad  HASSIO, come mi ha detto Diego le versioni e configurazioni sono personali quindi nessun consiglio, voglio solo informarti anche su una raspbian (SO della Rasperry) per desktop
Puoi seguire questa Guida https://github.com/indomus/forum/discussions/813

**Comment by stefano196710** [comment]
Grazie, per le guide sicuramente ci darò una lettura, ma adesso ho un problema più "grave".

Oggi, ho cambiato il router per migliorare la connessione in casa, ma è crollata tutta la domotica.
Con Chrome non riesco ad entrare in locale (homeassistant.local:8123), come se l'indirizzo fosse inesistente, il browser da errore.
Se uso Firefox funziona, ma la maggioranza dei dispositivi collegati non sono raggiungibili.
Oltretutto nel router non vedo il Raspberry tra i DHCP, quindi non riesco ad assegnarli un indirizzo IP statico.

Per la connessione dall'esterno, ho un IP pubblico fisso, quindi configurando 2 porte funzionava, ma ho già problemi in locale.

Come router prima avevo un TP-Link WR841N e adesso (per famigliarità d'utilizzo) un TP-Link Archer C50.

Cosa posso fare per risolvere?
Visti i problemi con i dispositivi a valle, si riesce a rendere la domotica più "Plug and Play ed indipendente", cioè avere meno disagi se cambio il modem?
Non ho cambiato configurazione ad Home Assistant che dovrebbe fare "da gestore centrale", ok che devo impostare il modem per raggiungerlo da fuori, ma assieme ho perso tutti i Sonoff e simili.

Scusate per le varie domande
Grazie

**Comment by stefano196710** [comment]
Ciao a tutti.
Dopo diverse prove e ore, riesco nuovamente a gestire HA in locale e tutti i dispositivi domotici (come Sonoff o Shelly) cercandoli tramite indirizzo IP.

Si riescono a gestire meglio tutti i dispositivi ed evitare questi problemi?
Grazie
